libs/program_options/src/variables_map.cpp

```diff
@@ -147,12 +147,12 @@ namespace hpx { namespace program_options {
         }
     }
 
-    HPX_EXPORT void store(const wparsed_options& options, variables_map& m)
+    void store(const wparsed_options& options, variables_map& m)
     {
         store(options.utf8_encoded_options, m, true);
     }
 
-    HPX_EXPORT void notify(variables_map& vm)
+    void notify(variables_map& vm)
```

Comment:
I'm sure that this symbol needs to be exported. Should we mark it as such at the declaration point instead? The same is likely true for the other symbols from this module.

Reply:
It already marked to be exported in the header file: https://github.com/STEllAR-GROUP/hpx/blob/62c825ff76ffc3a21588ca2c36730c9c9753aa5a/libs/program_options/include/hpx/program_options/variables_map.hpp#L68 (and the header file is included here).